data/resources/ui/pod/creation-page.ui

```diff
@@ -114,6 +163,49 @@
                                     <property name="orientation">vertical</property>
                                     <property name="spacing">18</property>
 
+                                    <child>
+                                      <object class="AdwPreferencesGroup">
+                                        <property name="title" translatable="yes">Devices</property>
+
+                                        <child>
+                                          <object class="GtkListBox" id="devices_list_box">
+                                            <style>
+                                              <class name="boxed-list"/>
+                                            </style>
+                                          </object>
+                                        </child>
+
+                                      </object>
+                                    </child>
+
+                                    <child>
+                                      <object class="AdwPreferencesGroup">
+                                        <property name="title" translatable="yes">Hosts</property>
+
+                                        <child>
+                                          <object class="AdwActionRow" id="disable_hosts_row">
+                                            <property name="activatable_widget">disable_hosts_switch</property>
+                                            <property name="title" translatable="yes">Disable Hosts management</property>
+                                            <child>
+                                              <object class="GtkSwitch" id="disable_hosts_switch">
+                                                <property name="valign">center</property>
+                                                <property name="action-name">pod.toggle-hosts</property>
+                                              </object>
+                                            </child>
+                                          </object>
+                                        </child>
```

Comment:
It looks better if we use a switch in the header suffix. I would also change the behavior in a way that the hosts become enabled if the switch is active instead of disable it.

![grafik](https://user-images.githubusercontent.com/3630213/193688552-41c38116-5f63-4509-ac49-51a2c70957e5.png)

```suggestion
                                        <property name="header-suffix">
                                          <object class="GtkBox">
                                            <property name="spacing">6</property>

                                            <child>
                                              <object class="GtkLabel">
                                                <property name="label" translatable="yes">Enabled</property>
                                              </object>
                                            </child>

                                            <child>
                                              <object class="GtkSwitch" id="enable_hosts_switch">
                                                <property name="valign">center</property>
                                                <property name="action-name">pod.toggle-hosts</property>
                                              </object>
                                            </child>

                                          </object>
                                        </property>
```

Reply:
That is actually what I was aiming for initially but I'm just very unfamiliar with GTK :)